templates/Includes/MainNav.ss

```diff
@@ -1,36 +1,32 @@
 <div class="main-nav">
```

Comment:
Would be good to append container in this class, as the nav is currently sitting on the left side of the screen :)